pkgs/applications/misc/fuzzel/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, fetchgit, pkgconfig, meson, ninja, wayland, pixman, cairo, librsvg, wayland-protocols, wlroots, libxkbcommon, scdoc, git, tllist, fcft}:
+
+stdenv.mkDerivation rec {
+  pname = "fuzzel";
+  version = "1.3.0";
+
+  src = fetchgit {
+    url = "https://codeberg.org/dnkl/fuzzel";
+    rev = "${version}";
+    sha256 = "12jv5iwmksygw8nfkxbd9rbi03wnpgb30hczq009aqgy7lyi5zmp";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson ninja scdoc git ];
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.